contact.html

```diff
@@ -0,0 +1,131 @@
+                                <li role="presentation"class="active"><a href="#">home</a></li>
+                                <li role="presentation"><a href="#">products</a></li>
+                                <li role="presentation"><a href="#">contact us</a></li>
+                            </ul>
+                        </nav>
+                        <!-- end Navebar section -->
+                    </header>
+        </div>
+        
+<div class="container content">
+    <div class="row">
+        <div class="col-sm-12 col-md-6">
+            <div class="row">
+                <div class="col-xs-6 col-sm-4"><img src="img/doctor1.jpg" alt="" class="img-responsive"></div>
+                <div class="col-xs-6 col-sm-8">
+                    <h3>Dr.Smith</h3>
+                </div>
+                <div class="col-xs-12 col-sm-8">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima, est inventore nam iusto nisi facere harum vitae ratione! Ratione et distinctio eaque ut deserunt culpa.</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-xs-6 col-sm-4"><img src="img/doctor2.jpg" alt="" class="img-responsive"></div>
+                <div class="col-xs-6 col-sm-8">
+                    <h3>Dr.Smith</h3>
+                </div>
+                <div class="col-xs-12 col-sm-8">
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Obcaecati at, fugiat ea accusamus quas magnam dolore natus laudantium! Provident, illum hic? Necessitatibus, voluptates. Ipsa, expedita!</p>
+                </div>
+            </div>
+            <div class="row">
+                <table class="table table-striped">
+                    <thead>
+                        <tr>
+                            <th></th>
+                            <th></th>
+                            <th></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td></td>
+                            <td>834-987-0989</td>
+                            <td>x890</td>
+                        </tr>
+                        <tr>
+                            <td></td>
+                            <td>834-987-0989</td>
+                            <td>x890</td>
+                        </tr>
+                        <tr>
+                            <td></td>
+                            <td>834-987-0989</td>
+                            <td>x890</td>
+                        </tr>
+                        <tr>
+                            <td></td>
+                            <td>834-987-0989</td>
+                            <td>x890</td>
+                        </tr>
+                        <tr>
+                            <td></td>
+                            <td>834-987-0989</td>
+                            <td>x890</td>
```

Comment:
This table is incomplete. Normally. this is not an acceptable merge due to incompletion but due to time, we will override for now. As soon as you can, i would create a branch to fix this.